From b7005a79dde509aa8226c42965b3e86aea93dc1f Mon Sep 17 00:00:00 2001
From: Ivan Iziumov <iiziumov@anaconda.com>
Date: Fri, 17 Oct 2025 12:28:06 +0200
Subject: [PATCH] patch windows path

---
 tests/test_config.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/tests/test_config.py b/tests/test_config.py
index 6cf9522..92fde66 100644
--- a/tests/test_config.py
+++ b/tests/test_config.py
@@ -196,7 +196,7 @@ def test_settings_toml_error(monkeypatch: MonkeyPatch, tmp_path: Path) -> None:
     with pytest.raises(AnacondaConfigTomlSyntaxError) as excinfo:
         _ = DerivedSettings()
 
-    assert "/config.toml: Unclosed array (at line 3, column 1)" in excinfo.value.args[0]
+    assert "\\config.toml: Unclosed array (at line 3, column 1)" in excinfo.value.args[0]
 
 
 def test_settings_validation_error(monkeypatch: MonkeyPatch, tmp_path: Path) -> None:
@@ -217,8 +217,8 @@ def test_settings_validation_error(monkeypatch: MonkeyPatch, tmp_path: Path) ->
         _ = DerivedSettings(not_required=3)  # type: ignore
 
     message = excinfo.value.args[0]
-    assert "/config.toml in [plugin.derived] for foo = 1" in message
-    assert "/config.toml in [plugin.derived] for nested.field = [0, 1, 2]" in message
+    assert "\\config.toml in [plugin.derived] for foo = 1" in message
+    assert "\\config.toml in [plugin.derived] for nested.field = [0, 1, 2]" in message
     assert (
         "Error in environment variable ANACONDA_DERIVED_OPTIONAL=not-an-integer"
         in message
@@ -275,13 +275,13 @@ def test_error_handled(
 
     result = invoke_cli(["config-error", "syntax-error"])
     assert result.exit_code == 1
-    assert "/config.toml: Unclosed array (at line 3, column 1)" in result.stdout
+    assert "\\config.toml: Unclosed array (at line 3, column 1)" in result.stdout
 
     result = invoke_cli(["config-error", "validation-error"])
     assert result.exit_code == 1
-    assert "/config.toml in [plugin.derived] for foo = 1" in result.stdout
+    assert "\\config.toml in [plugin.derived] for foo = 1" in result.stdout
     assert (
-        "/config.toml in [plugin.derived] for nested.field = [0, 1, 2]" in result.stdout
+        "\\config.toml in [plugin.derived] for nested.field = [0, 1, 2]" in result.stdout
     )
     assert (
         "Error in environment variable ANACONDA_DERIVED_OPTIONAL=not-an-integer"
-- 
2.50.1 (Apple Git-155)

